.ci/jobs/defaults.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+##### GLOBAL METADATA
+
+- meta:
+    cluster: apm-ci
+
+##### JOB DEFAULTS
+
+- job:
+    view: APM-CI
+    project-type: multibranch
+    logrotate:
+      daysToKeep: 30
+      numToKeep: 100
+    number-to-keep: '5'
+    days-to-keep: '1'
+    concurrent: true
+    node: linux
+    script-path: .ci/Jenkinsfile
+    scm:
+    - github:
+        branch-discovery: no-pr
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: true
+        repo: ecs-dotnet
+        repo-owner: elastic
+        credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        build-strategies:
+        - tags:
+            ignore-tags-older-than: -1
+            ignore-tags-newer-than: -1
+        - regular-branches: true
+        - change-request:
+            ignore-target-only-changes: false
+        clean:
+          after: true
+          before: true
+        prune: true
+        shallow-clone: true
+        depth: 3
+        do-not-fetch-tags: true
+        submodule:
```

Comment:
Do you think it would be good to add a `reference-repo` here?

Reply:
Good point!
The repo is not as big as other git repos and the number of CI builds might not be too much, so I'd say we don't need it for now. and if we need it then we need to add this repo to the list of the reference repos supported in the CI workers.

Reply:
Works for me. 👍